src/context/AppProvider/AppContext.tsx

```diff
@@ -119,10 +121,16 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
             ),
           )
         })
+
+        dispatch(portfolio.actions.setIsAccountsLoading(false))
```

Comment:
We could probably clean this up with a try/finally and just have the one call to set to false in the finally.

This first `hasManagedAccounts` condition also doesn't await the `getAccount.initiate` so to make a distinction between portfolio and accountMetatdata we could clarify as `isLoadingAccountMetadata` as an option

Reply:
You are right, used a try/finally and updated everything to `isAccountMetadataLoading` in [e72bded](https://github.com/shapeshift/web/pull/7629/commits/e72bded769397fe997646cd23ed7ff15a20d7bc6)